indra/tests/test_pubchem_client.py

```diff
@@ -4,3 +4,9 @@
 def test_get_inchi_key():
     ik = pubchem_client.get_inchi_key('5280613')
     assert ik == 'JLVSPVFPBBFMBE-HXSWCURESA-O', ik
+
+
+def test_get_pmids():
```

Comment:
add a test for an obviously wrong PMID and also for an okay looking one that's invalid